apps/src/templates/Button.jsx

```diff
@@ -176,7 +177,7 @@ class Button extends React.Component {
         tabIndex={tabIndex}
         id={id}
       >
-        <div>
+        <div style={_.pick(style, ['textAlign'])}>
```

Comment:
This is a workaround for an unfortunate issue where this CSS rule trying to make teacher dashboard (on pegasus) use dashboard styles, was coercing `Button` contents to be left-aligned.

![image](https://user-images.githubusercontent.com/1615761/42538863-05f73a5c-844e-11e8-92b0-b4047c441737.png)

IMO that's much too broad of a CSS rule, but I'm not sure what the ramifications of removing it are and it seems appropriate to make the button contents respect any `text-align` we give it anyway.